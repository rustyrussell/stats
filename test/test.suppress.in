same number 1234
from 1 to 100

same number 1234
from 1 to 200

Lone line. - Also note the blank lines.
