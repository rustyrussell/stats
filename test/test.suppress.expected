from 1 to 100-200(150+/-50)
